### PR TITLE
Add recommendation categories

### PR DESCRIPTION
**Pull request by faihegberg**
Add recommendation categories to accommodate new recommendation content, as per https://trello.com/c/e7c6H4YW/228-update-recommendations-module

### CODE REVIEW

File: src/common/utils/recommend.ts
Comment (mikelehen) [inline src/common/utils/recommend.ts:103]: This is fine since it's following the existing pattern, but it's getting pretty repetitive, which makes it harder to read and more likely for somebody to make mistakes when changing it.  I think we could restructure this to be easier to read / more maintainable by turning this into a for loop something like:

```
const recommendationCategoryOrdering = [
  RecommendCategory.MASKS,
  RecommendCategory.GATHERINGS,
  ...
];

const remainingRecommendations = recommendations;
const orderedRecommendations: Recommendation[] = [];
for (const category of recommendationCategoryOrdering) {
  const [inCategory, notInCategory] = partition(remainingRecommendations, item => item.category === category);
  orderedRecommendations = orderedRecommendations.concat(inCategory);
  remainingRecommendations = notInCategory;
}
```

That said, if you don't want to refactor this code, that's fine.  Maybe add a:

```
// TODO(fai): Consider refactoring this repetitive partition() code into a for loop or similar.
```

If you don't feel comfortable with your name on it, feel free to do TODO(michael). 😁

**Review by mikelehen — APPROVED** [review]
Looks good.  I think this code could be simplified with some refactoring, but feel free to keep as-is for now.  If you do refactor and want another review, I'm happy to take another look at it.

File: src/common/utils/recommend.ts
Comment (faihegberg) [reply src/common/utils/recommend.ts]: I agree. I was thinking it's getting repetitive as I was adding three more categories into this 😆 . I implemented what you suggested.

File: src/common/utils/recommend.ts
Comment (faihegberg) [reply src/common/utils/recommend.ts:69]: Yes, I left this out intentionally cos I want to confirm with the content team today regarding the order :) Once confirmed, I'll add them and merge.

File: src/common/utils/recommend.ts
Comment (faihegberg) [reply src/common/utils/recommend.ts:85]: Actually, I just checked with Mikayla and the exposure app recommendation and the rest of the recommendation items not in the [mocks](https://www.figma.com/file/AwKgjN0Q4cgt3hSyF1I5FH/2022?node-id=86%3A521) are getting removed altogether.

**Comment by faihegberg** [comment]
[Update recommendations module](https://trello.com/c/e7c6H4YW/228-update-recommendations-module)

**Pull request merged**
